exp/bufferpool/pool.go

```diff
@@ -4,7 +4,8 @@ package bufferpool
 import "sync"
 
 const (
-	nBuckets = 20
+	minSize  = 1024
+	nBuckets = 11 // max 1Mb
```

Comment:
Maybe spell out the math behind this a bit more, or (better, if you can figure out how) somehow define it as an expression.

Also, nit: the unit you're looking for is MiB.

Also, I wish Go had static assertions.

Reply:
>Also, nit: the unit you're looking for is MiB.

Thanks - for some reason, I can never remember this 😆